ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -97,6 +97,14 @@
     dest: "{{ jenkins.dest }}"
     mode: "{{ jenkins.mode|default('0644') }}"
 
+# TODO - Should this run on every machine?
```

Comment:
There is already config eslsewhere that runs/configures what is needed to start the jenkins agent.  Look in roles/jenkins-worker/vars/main.yml and related which sets up which scripts are run to start the jenkins agent.

Reply:
Yep, but we used to always have `~/start.sh` as well, question here is whether we still need start.sh if we have other startup methods.

I'd suggest we do, because it gives us a consistent way to restart machines, without having to remember all the different commands for different architectures.

Reply:
It is worth noting that we need start.sh for the mac machines to start automatically. The question is whether we want this on other platforms too

Reply:
I think we used to have start.sh only because we had not investigated/implemented the more natural way to do the same thing for each os flavor.